src/libtcod/renderer_xterm.c

```diff
@@ -159,7 +175,257 @@ static void xterm_destructor(struct TCOD_Context* __restrict self) {
   xterm_cleanup();
 }
 
-TCOD_Context* TCOD_renderer_init_xterm(const char* window_title) {
+static void send_sdl_key_press(SDL_Keycode ch, bool shift) {
+    bool is_ascii = ch <= (SDL_Keycode)INT_MAX && isascii(ch);
```

Comment:
If a variable won't be changed later then it should be `const` to be easier to read.
```suggestion
    const bool is_ascii = ch <= (SDL_Keycode)INT_MAX && isascii(ch);
```